### DISCUSSION TITLE
DNS based Reverse Shell

### DISCUSSION THREAD

**Discussion by sfunkernw** [projectdiscovery/interactsh#387] [General]
Hi,

would it be possible to use the interact.sh self-hosted domain also for a DNS based reverse shell?
I found this project https://github.com/sysdream/chashell and wanted to give it a spin. 
However, as my self-hosted domain is having the NS1 and NS2, I can't configure those entries in GoDaddy (my domain management vendor), I am not sure where I could configure the required DNS entries:

```
chashell 300 IN A [SERVERIP]
c 300 IN NS chashell.[DOMAIN].
```

I guess this could be done somewhere on my self-hosted server?

**Comment by MetzinAround** [comment]
Closing this old issue. Feel free to reopen if you like!
